### PR TITLE
Move stats radio button facet to the top

### PR DESCRIPTION
**Pull request by sihugh**
At present, it looks like the document type field is under the "Topic" heading
in the filters section.  This fixes that, and also makes it nearer the top
on mobile.  It's more used than Topic on the stats finder, so this makes sense.

We'd originally kept topic as the top filter to keep things broadly consistent
between supergroup finders as we expected to build some navigation to allow
users to flip between them.  This hasn't happened, so let's do a small
thing to make life easier for people.

## Before 

Screenshot 2019-12-18 16 35 20

## After

Screenshot 2019-12-18 16 35 28

### CODE REVIEW

File: lib/learn_to_rank/ranker.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@ def tensorflow_serving_ip
     end
 
     def ranker_error(response)
-      response.nil? || response.code != 200
+      response.body.nil? || response.body.empty? || response.code != 200
```
Comment (laurentqro) [inline lib/learn_to_rank/ranker.rb]: I think `response.body.nil? || response.body.empty?` could be replaced with `response.body.blank?`

```
Loading development environment (Rails 5.2.4)
[1] pry(main)> nil.blank?
=> true
[2] pry(main)> " ".blank?
=> true
[3] pry(main)> "".blank?
=> true
```

**Comment by laurentqro** [comment]
@sihugh small comment, otherwise 🚢

**Review by laurentqro — APPROVED** [review]

File: lib/learn_to_rank/ranker.rb (hunk 1 above)
Comment (sihugh) [reply lib/learn_to_rank/ranker.rb]: I'd thought that as search-api is Sinatra, it wouldn't have `blank?`.  Turns out we have ActiveSupport added so I can use it.  Thanks

Comment (laurentqro) [reply lib/learn_to_rank/ranker.rb]: Ah yes, actually hadn't realised that. Great!

**Pull request merged**